### PR TITLE
Bump detekt-version to 1.8.0

### PR DESCRIPTION
**Pull request by pinkasey**
bump version to 1.8.0, and marked "check" as thread-safe

closes https://github.com/Ozsie/detekt-maven-plugin/issues/42
closes https://github.com/Ozsie/detekt-maven-plugin/issues/34

### CODE REVIEW

File: src/main/java/com/github/ozsie/CheckMojo.kt [hunk 1]
```diff
@@ -12,14 +12,18 @@ import java.nio.file.Paths
 @Suppress("unused")
 @Mojo(name = "check",
         defaultPhase = LifecyclePhase.VERIFY,
+        threadSafe = true,
```
Comment (pinkasey) [inline src/main/java/com/github/ozsie/CheckMojo.kt:15]: @Ozsie 
@arturbosch
I take your word that it's threadsafe -
I dug a little into the code but didn't get far
so I base my assumption on https://github.com/Ozsie/detekt-maven-plugin/issues/34

I ran it in parallel on my maven project, and it worked fine.
(`mvn detekt:check -T 4 `)

File: src/main/java/com/github/ozsie/CheckMojo.kt [hunk 2]
```diff
@@ -12,14 +12,18 @@ import java.nio.file.Paths
...
 class CheckMojo : DetektMojo() {
     override fun execute() {
         getCliSting().forEach {
             log.debug("Applying $it")
         }
-        val cliArgs = parseArguments<CliArgs>(getCliSting().log().toTypedArray())
+        val cliArgs = parseArguments<CliArgs>(getCliSting().log().toTypedArray(), System.out, System.err)
         skip = !Files.isDirectory(Paths.get(input))
-        if (!skip) return Runner(cliArgs).execute() else log.info("Input directory '$input' not found, skipping module")
+        if (!skip)
```
Comment (pinkasey) [inline src/main/java/com/github/ozsie/CheckMojo.kt:24]: this line was too long after I updated it
and frankly - it wasn't very readable as one-liner

File: src/test/java/com/github/ozsie/CheckMojoSpec.kt [hunk 3]
```diff
@@ -21,7 +21,6 @@ class CheckMojoSpec : Spek({
     given("a CheckMojo and 'skip' is false") {
         val checkMojo = CheckMojo()
         checkMojo.skip = false
-        checkMojo.input = "."
```
Comment (pinkasey) [inline src/test/java/com/github/ozsie/CheckMojoSpec.kt:24]: Test failed so I removed this line.
The default value is `"\${basedir}/src"`

**Comment by Ozsie** [comment]
Awesome, I'll merge this in now and create a release in a few hours, as soon as I have time

**Comment by pinkasey** [comment]
@Ozsie 
Great thanks!
If it's not too much trouble - I'd appreciate if you post a comment after this is available for consumption

**Comment by Ozsie** [comment]
@pinkasey It should be available from maven central now

**Comment by pinkasey** [comment]
Thanks!
It works

**Pull request merged**